domains/crvt.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "portfolio / profile page",
+    "repo": "https://github.com/slovakians/crvts",
+    "owner": {
+        "username": "Slovakians",
+        "email": "siresirol937@gmail.com",
+        "twitter": "<twitter-username>"
```

Comment:
```suggestion
```
Either put your Twitter username here or delete the line entirely.

Reply:
oh opps wrong link